src/main/java/net/minecraftforge/common/world/StructureSpawnManager.java

```diff
@@ -99,7 +99,7 @@ private static void gatherEntitySpawns(Map<Structure<?>, StructureSpawnInfo> str
             Structure<?> structure = entry.getKey();
             StructureSpawnInfo spawnInfo = entry.getValue();
             //Note: We check if the structure has spawns for a type first before looking at the world as it should be a cheaper check
-            if (spawnInfo.spawns.containsKey(classification) && structureManager.getStructureStart(pos, spawnInfo.insideOnly, structure).isValid())
+            if (spawnInfo.spawns.containsKey(classification) && structureManager.func_235010_a_(pos, spawnInfo.insideOnly, structure).isValid())
```

Comment:
This looks like you are building against the version from before mappings got updated a few days ago, you should update forgedev as I am pretty sure this change will cause it not to build properly if merged.

Reply:
Fix